docs/graph/providers/InteractiveProviderBehavior.md

```diff
@@ -9,7 +9,9 @@ dev_langs:
 
 # InteractiveProviderBehavior XAML Behavior
 
-<!-- Describe your control -->
+> [!WARNING]
+> (This API has been removed. Please use the official [Microsoft Graph SDK](https://github.com/microsoftgraph/msgraph-sdk-dotnet) directly. For the latest guidance on using the Microsoft Graph in the Toolkit check out the [Windows Community Toolkit - Graph Helpers and Controls](../overview.md).)
```

Comment:
The base Graph SDK doesn't quite replicate this scenario, so do we just want the 2nd sentence to point folks to our starting point?
```suggestion
> (This API has been removed. For the latest guidance on using the Microsoft Graph in the Toolkit check out the [Windows Community Toolkit - Graph Helpers and Controls](../overview.md).)
```

We could put this full warning instead on the old Microsoft Graph Service page?

Reply:
FYI in case you missed this one @shweaver-msft

Reply:
Fixed :)